tests/run-tests.py

```diff
@@ -115,6 +200,53 @@ def convert_regex_escapes(line):
     return bytes("".join(cs), "utf8")
 
 
+def get_test_instance(test_instance, baudrate, user, password):
+    if test_instance.startswith("port:"):
```

Comment:
It's not really clear to me where/how this can be specified, should this be in the help (-h) output?

Reply:
Ah, I missed documenting this.  Now added to the help text.